packages/mux-player/REFERENCE.md

```diff
@@ -31,6 +31,7 @@
 | `playback-token`              | `string`                                                                                                    | The playback token for signing the `src` URL.                                                                                                                                                                                                                                                                                                                                                    | N/A           |
 | `thumbnail-token`             | `string`                                                                                                    | The thumbnail token for signing the `poster` URL.                                                                                                                                                                                                                                                                                                                                                | N/A           |
 | `storyboard-token`            | `string`                                                                                                    | The storyboard token for signing the storyboard URL.                                                                                                                                                                                                                                                                                                                                             | N/A           |
+| `storyboard-src`            | `string` (URL)                                                                                                    | Full URL string for the storyboard asset. Typically derived from the `playbackId`, setting this attribute will overide the derived storyboard.                                                                                                                                                                                                                                                                                                                                              | `undefined`           |
```

Comment:
Should mention here that the format of the storyboard-src is a .vtt and link to an example:

```
https://image.mux.com/FuJSYrK0014ec2LPnm11bzC2MAySAQPqA/storyboard.vtt
```